charts/incubator/channels-dvr/values.yaml

```
@@ -0,0 +1,27 @@
+image:
+  repository: fancybits/channels-dvr
+  pullPolicy: Always
+  tag: latest@sha256:5cad00679feb37533f0eccf72b70c3bf7acc4b4c18fcae65e2081e7222da39ce
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 8089
+        targetPort: 8089
```

```suggestion
        port: 8089
        protocol: HTTP
        targetPort: 8089
```

If that port is for the webui, it's probably HTTP :)